creator/events/migrations/0022_add_referral_token.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.26 on 2022-01-27 20:22
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('events', '0021_add_template_download'),
```

Comment:
```suggestion
        # Need to force load the referral_tokens app by making dependency on
        # latest migration. If we don't do this the migration fails bc it
        # cannot find the related model, referral_tokens.ReferralToken
        # https://stackoverflow.com/questions/33496333/valueerror-related-model-uapp-model-cannot-be-resolved
        ('referral_tokens', '__latest__'),
        ('events', '0021_add_template_download'),
```

The latest error you're having is bc django can't find/understand the related model `referral_tokens.Referral_Token`. I found a fix on stack overflow which makes django load the Referral_Token model as a dependency of this migration. See comments above. This should fix the issue